pkg/install/install.go

```diff
@@ -268,7 +268,7 @@ func (i *Installer) getBlobService(ctx context.Context) (*azstorage.BlobStorageC
 func (i *Installer) loadGraph(ctx context.Context) (graph, error) {
 	i.log.Print("load graph")
 
-	blobService, err := i.getBlobService(ctx)
+	blobService, err := i.getBlobService(ctx, mgmtstorage.Permissions("crw"))
```

Comment:
If you end up doing this, I guess you can refine the permissions asked for, e.g. no "cw" when you're reading, etc.